website/screens/components/date-input/code/examples/basic.ts

```diff
@@ -0,0 +1,32 @@
+import { DxcDateInput, DxcStack, DxcInset } from "@dxc-technology/halstack-react";
+
+const code = `() => {
+    return(
+        <DxcInset space="large">
+            <DxcStack gutter="small">
+                <DxcDateInput
+                label="Default"
+                placeholder="Placeholder"
+                />
+                <DxcDateInput
```

Comment:
Remove this example, since this should be on a `Usage` tab showcase.

Reply:
I removed the examples with error and disabled and only left the most basic one.